website/pages/en/index.js

```diff
@@ -137,11 +140,20 @@ class Index extends React.Component {
 
     const FeatureCallout = () => (
       <div  className="featureShowcaseSection  paddingBottom" style={{textAlign: 'center'}}>
-        <h2>Use Cases</h2>
-        <MarkdownBlock>{`Document business [use cases](${docUrl('use-cases/overview')}) that drive FDC3 interoperability standards.`}</MarkdownBlock>
+        <h2>FDC3 Survey</h2>
```

Comment:
Does this change get rid of the use cases banner? Not sure we want that... Ideally a second banner?

Reply:
It's a temporary change to highlight the survey for the next week (right?). I just hid the Markdown block so we can put it back up as soon as survey is done. I was asked to get this up quickly to help with draw attention to the survey. I tried a couple of other ways at first, but kept having issues getting them to work.

Reply:
Use Cases are still available in the top toolbar so given timing here - I am ok with temporarily removing the use case banner.

Thanks,

Johan